docs/deployment/quick-setup.md

```diff
@@ -33,7 +33,8 @@ variables.
 
 **Step 6:** Run docker-compose
 
-    docker compose -f docker-compose.yml -f docker-compose.dev.yml -p aaq-stack up -d --build
+    docker compose -f docker-compose.yml -f docker-compose.dev.yml \
+        -p aaq-stack up -d --build
```

Comment:
Is this slash necessary? Might just be a bit confusing - a scrollable element is fine isn't it?

Reply:
Personal preference. I don't like the side scroll and would rather see it all in one glance.